message-markdown.md

```diff
@@ -0,0 +1,51 @@
+# Markdown in messages
+> This list is for every formating thing that goes beyond plain-text.
+
+Deltachat is going to support the following subset of markdown:
+
+### `*italics*`
+
+### `**bold**`
+
+### `~~strikethrough~~`
+
+### `https://delta.chat` - Urls
+
+Make URLs clickable, but **don't** support the Link syntax `[Name](url)`
+because that could be used for misleading and as such be used in scams.
+
+### `` `inline-code` ``
+useful to send non markdown text in your message like source code snippets.
+Should get rendered in a monospace font and with a different background.
+ 
+### ` ``` fence code block ``` `
+
+```
+Similar to `inline-code` but not inline and it may supports code highlighting.
+```
+` ```[lang?] [content]``` `
+ A bit modified from the common syntax to allow one liners.
+ 
+The code **highlighting** is **optional** as implementation (time)cost 
+may not be worth the small gain.
+The `language` definition should be parsed separately and omitted in this case.
+
+### `:emoji:`
```

Comment:
never got the point from these kind of emoji ... esp. as we already support utf-8-emojis directly. I would strike that for now.

Reply:
Its mainly that you can type an emoji without searching it in the loooong emoji keyboard.
Telegram or discord for example provide also auto completion with this syntax

Reply:
okay, that's a point :lach: mainly useful for desktop. however, maybe the type-in can be converted to a unicode character before sending.

Reply:
its also useful on mobile, try typing :+1 for example on telegram on android (not sure if ios has this feature too rightnow because I haven't used telegram ios in a while.)